lithops/serverless/backends/oracle_f/config.py

```diff
@@ -0,0 +1,66 @@
+# (C) Copyright Cloudlab URV 2023
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+from lithops.constants import TEMP_DIR
+
+
+DEFAULT_CONFIG_KEYS = {
+    'runtime_timeout': 300,  # Default: 5 minutes
+    'runtime_memory': 256,  # Default memory: 256 MB
+    'max_workers': 300,
+    'worker_processes': 1,
+    'invoke_pool_threads': 64,
+}
+
+CONNECTION_POOL_SIZE = 300
+
+APPLICATION_NAME = 'lithops'
+BUILD_DIR = os.path.join(TEMP_DIR, 'OracleRuntimeBuild')
+
+AVAILABLE_PY_RUNTIMES = {
+    '3.8': 'python3',
+}
```

Comment:
Is Oracle functions available only for python3.8? I can see you added a Dockefile, so: does this mean Oracle Functions can work with any Python version?